algorithms/CPlusPlus/Recursion/reverse-a-number.cpp

```diff
@@ -0,0 +1,34 @@
+/*
+Description: Program to reverse integer using recursion
+Time complexity: O(n) where n is the number of digits in the integer
+*/
+
+#include <iostream>
+#include <math.h>
```

Comment:
```suggestion
#include <cmath>
#include <iostream>
```

This one is causing the linting checks to fail:
prefer cpp style header and put c headers before cpp headers